src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -543,6 +543,15 @@ public void filterSensitiveInformation() {
         setGradingCriteria(null);
     }
 
+    /**
+     * Gets the type of the exercise as a string
+     *
+     * @return type of the exercise as a string
+     */
+    public String getStringRepresentationOfType() {
```

Comment:
I don't understand why we need this?
This would be sent on all json responses of all requests (and exercises are really used very often).
However the type is already available

Reply:
removed